schema/bidder_position.js

```diff
@@ -45,6 +45,15 @@ const BidderPositionType = new GraphQLObjectType({
       type: SaleArtwork.type,
       resolve: position => gravity(`sale_artwork/${position.sale_artwork_id}`),
     },
+    is_winning: {
```

Comment:
> Although I wonder if that's a bid deal depending on how the caching works. I also wonder if there's some kind of currying/lazy-loading thing we could put in gravity that can recognize the same query being made twice at the same time and make only one fetch.

That is exactly what happens actually. You should be able to debug the number of requests that's happening by looking in the console but, if you're using the gravity loader (which you are) to load a resource, then it will only be fetched a single time.